IBAnimatable/GradientDesignable.swift

```diff
@@ -78,16 +46,46 @@ fileprivate extension GradientDesignable where Self: UIView {
   }
 
   func configurePredefinedGradient() -> GradientColor? {
-
     guard let gradientType = predefinedGradient else {
       return nil
     }
     return gradientType.colors
   }
 }
 
+public extension GradientDesignable where Self: UINavigationBar {
+  public func configureGradient() {
+    let predefinedGradient = configurePredefinedGradient()
+    if let startColor = startColor, let endColor = endColor {
+      configureGradient(startColor: startColor, endColor: endColor)
+    } else if let startColor = predefinedGradient?.start, let endColor = predefinedGradient?.end {
+      configureGradient(startColor: startColor, endColor: endColor)
+    }
+  }
+
+  fileprivate  func configureGradient(startColor: UIColor, endColor: UIColor) {
```

Comment:
I agree for the `configureGradient`, I was talking about `configureGradient(startColor: UIColor, endColor: UIColor)` and `func makeGradientLayer() -> CAGradientLayer` (which are currently file private)